tests/warp/warp_test.go

```diff
@@ -200,9 +200,7 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 
 		packedInput, err := warp.PackSendWarpMessage(warp.SendWarpMessageInput{
-			DestinationChainID: common.Hash(blockchainIDB),
-			DestinationAddress: fundedAddress,
-			Payload:            payload,
+			Payload: payload,
```

Comment:
Should we remove SendWarpMessageInput as a struct since it's just []byte for payload now?

Reply:
I think this will make it future proof, but I also like the idea of just passing `[]byte`. It's ultimately your call @aaronbuchwald

Reply:
Yup, we should make it just `[]byte`. Could you re-generate the stateful precompile using the precompile gen tool and replace the packers/unpackers that are impacted by this change? One of them will be making this change to `[]byte`